GameSpy/client.go

```diff
@@ -172,21 +172,28 @@ func (client *Client) readFESL(data []byte) {
 		payloadType := string(payloadTypeRaw)
 
 		binary.Read(p, binary.BigEndian, &payloadId)
-		binary.Read(p, binary.BigEndian, &payloadLen)
+		
+		if binary.Len() < 4 {
+			outcommand.Message = make(map[string]string)
```

Comment:
Can you also set the outCommand.Query and outCommand.PayloadID just to avoid issues when trying to access that somewhere down the line? Looks awesome otherwise <3